deep_translator/constants.py

```diff
@@ -108,24 +109,146 @@
     'sv': 'swedish',
     'tg': 'tajik',
     'ta': 'tamil',
+    'tt': 'tatar',
     'te': 'telugu',
     'th': 'thai',
     'tr': 'turkish',
+    'tk': 'turkmen',
     'uk': 'ukrainian',
     'ur': 'urdu',
+    'ug': 'uyghur',
     'uz': 'uzbek',
     'vi': 'vietnamese',
     'cy': 'welsh',
     'xh': 'xhosa',
     'yi': 'yiddish',
     'yo': 'yoruba',
     'zu': 'zulu',
-    'fil': 'Filipino',
-    'he': 'Hebrew'
 }
 
 GOOGLE_LANGUAGES_TO_CODES = {v: k for k, v in GOOGLE_CODES_TO_LANGUAGES.items()}
 
+# This dictionary maps the primary name of language to its secondary names in list manner (if any)
+GOOGLE_LANGUAGES_SECONDARY_NAMES = {
+    'myanmar': ['burmese'],
+    'odia': ['oriya'],
+    'kurdish':  ['kurmanji']
+}
+
+test_text_standard = 'Hello world.'
+
+TRANSLATED_RESULTS = {
```

Comment:
Should I move 
```
test_text_standard
TRANSLATED_RESULTS
```
 to test_google_trans.py file?

Actually, I randomly test any five cases that's what I am trying to do as we know there are around 108 languages so test file will run any 5 cases to validate it.

Reply:
@kuspia creating a new file is better ;)

Reply:
Hi 
I removed test data from constants.py 
tried placing test_data.py inside tests 
```
from test_data import test_text_standard, TRANSLATED_RESULTS
```

I met with the following error:
```
E   ModuleNotFoundError: No module named 'test_data'
```

This happened since test_data is not a part of igel package and I can't import this way.


Solution: Either I can put the test_data in test_google_trans file or do you have any better suggestions,I thought creating test_data would be nice since it may be used by other translators.